addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -2237,11 +2237,7 @@ export function getFontSizeDisplayValue(sel, getCSSVariableValue, convertNumeric
         }
         remValue = parseFloat(getCSSVariableValue(`${fsName}-font-size`));
     }
-    // It's default font size (no font size class / style).
-    if (remValue === undefined) {
-        remValue = parseFloat(getCSSVariableValue("font-size-base"));
-    }
-    const pxValue = convertNumericToUnit(remValue, "rem", "px");
+    const pxValue = remValue && convertNumericToUnit(remValue, "rem", "px");
```

Comment:
I don't think this is the real issue. I think its because we don't get `closestFontSizedEl` for tags with class `lead` can you check if adding `lead` to `styleClassesRelatedToFontSize` works or not or maybe use `TEXT_STYLE_CLASSES` instead of `styleClassesRelatedToFontSize`.

Reply:
In the case of elements having `lead` class it will work but for the cases of elements not having lead or similar classes it will give wrong output since it will not find any class e.g. `btn-lg`. Hence IMO we should keep both of the fixes. I've updated the code, thanks!